### PR TITLE
[script.simkl] v1.0.0

### PR DESCRIPTION
**Pull request by daviddavo**
### Description

Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into.
### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] Each add-on submission should be a single commit with using the following style: [sciprt.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: script.simkl/addon.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <description lang="en_gb">Automatically set Movies and TV Shows\Anime as watched on Simkl.com. Simkl knows what you're watching, tells you episodes you've missed, and connects you to what your friends are into.</description>
+    <platform>all</platform>
+    <license>GPLv3</license>
+    <email>david@ddavo.me</email>
+    <source>https://github.com/SIMKL/script.simkl</source>
+    <assets>
+      <icon>resources/icon.png</icon>
+      <fanart>resources/fanart.png</fanart>
+    </assets>
+    <language>en es</language>
```
Comment (enen92) [inline script.simkl/addon.xml]: I think you should leave the languages tag blank. This is only applied to the content the plugin or script provides. Your addon.xml does not include a <provides> tag, so you're not providing media content with this script

File: script.simkl/service.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+from resources.lib import interface, engine
+from resources.lib import simklapi as API
+
+__addon__ = xbmcaddon.Addon()
+interface.__addon__ = __addon__
+autoscrobble = __addon__.getSetting("autoscrobble")
+def getstr(strid): return __addon__.getLocalizedString(strid)
+
+try:
+    with open(xbmc.translatePath(__addon__.getAddonInfo("path")).decode("utf-8") +"/resources/data/compdate.txt", "r") as f:
```
Comment (enen92) [inline script.simkl/service.py]: please use os.path.join() here

**Comment by daviddavo** [comment]
Both fixed

File: script.simkl/addon.xml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="1.0.0"
+       provider-name="David Davó / Simkl">
+  <requires>
+    <import addon="xbmc.python" version="2.6"/>
+    <import addon="script.module.simplejson"/>
+    <import addon="script.module.xbmcutil"/>
```
Comment (MartijnKaijser) [inline script.simkl/addon.xml]: You must specify the minimum version.

File: script.simkl/addon.xml [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.simkl"
+       name="Simkl TV Tracker"
+       version="1.0.0"
+       provider-name="David Davó / Simkl">
+  <requires>
+    <import addon="xbmc.python" version="2.6"/>
+    <import addon="script.module.simplejson"/>
```
Comment (MartijnKaijser) [inline script.simkl/addon.xml]: simplejson isn't really needed as it only had a speed gain with many JSON operations for python 2.6. In Kodi v17 all platforms us python 2.7 and it's no longer needed.

File: script.simkl/resources/language/English/strings.po [hunk 5]
```diff
@@ -0,0 +1,103 @@
+# Kodi Media Center language file
```
Comment (MartijnKaijser) [inline script.simkl/resources/language/English/strings.po]: Please move language file according to following folder structure http://kodi.wiki/view/language_support#Layout_of_the_language_folders

**Comment by daviddavo** [comment]
All fixed

File: script.simkl/service.py [hunk 6]
```diff
@@ -16,12 +16,12 @@
...
 def getstr(strid): return __addon__.getLocalizedString(strid)
 
-#try:
-compdatefile = os.path.join(__addon__.getAddonInfo("path").decode("utf-8"), "resources/data/compdate.txt")
-with open(xbmc.translatePath(compdatefile), "r") as f:
-    __compdate__ = f.read()
-#except:
-#    __compdate__ = "ERROR: No such file or directory"
+try:
+    compdatefile = os.path.join(__addon__.getAddonInfo("path").decode("utf-8"), "resources/data/compdate.txt")
```
Comment (enen92) [inline script.simkl/service.py]: can you please use join for all the nodes? `os.path.join(.........,"resources","data","compdate.txt")`

**Comment by daviddavo** [comment]
Btw, I fixed a bug so it works with python 2.6 (So on android too)

**Comment by enen92** [comment]
@daviddavo are you going to update the description or are you happy with the current state of the PR? (to know if I merge it right now or wait for any change)

**Comment by daviddavo** [comment]
The disclaimer is fine. You can merge it

**Comment by enen92** [comment]
thanks

**Pull request merged**